codecov.yml

```diff
@@ -14,3 +14,11 @@ coverage:
 comment:
   layout: "reach, diff, flags, files"
   behavior: default
+
+ignore:
+  - "test/*"
```

Comment:
Suggest to add code comments to explain why it could ignore unitest for these code. And other developers will not add the meaningless code